package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "react-native-helpscout-beacon",
   "title": "React Native Helpscout Beacon",
-  "version": "0.5.0",
+  "version": "0.6.0",
   "description": "Integrate Helpscout Beacon in to your React Native app",
-  "main": "lib/index.js",
-  "types": "lib/types.d.ts",
```

Comment:
Are you sure you should remove the `lib`? Did you change anything to warrant this change? It looks to me like the package might stop working if the consumer of the package doesn't transpile the TS themselves.

Reply:
We don't have a `lib` folder in the package present so this is pointing now to the right file. :)